valhalla/thor/dijkstras.h

```diff
@@ -78,22 +78,64 @@ class Dijkstras {
    * @param  mode_costing List of costing objects
    * @param  mode         Travel mode
    */
-  virtual void Compute(google::protobuf::RepeatedPtrField<valhalla::Location>& origin_locs,
```

Comment:
can you move these back to the `cc` file and use explicit template instantiations? i like hiding the details of the algorithms in the src files if we can. you can then also remove your `std::enable_if` because the explicit instantiation takes the place of it. or is there an other reason to have it in the header that im missing?

Reply:
No, just expedience - will move.